kitsune/messages/jinja2/messages/inbox.html

```diff
@@ -16,11 +16,12 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
     <form action="{{ url('messages.bulk_action') }}" method="POST">
       <div class="sumo-button-wrap">
         <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
+         <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
       </div>
       <ol class="inbox-table">
             <li class="email-row header">
-                <div class="email-cell checkbox">
-                    <input type="checkbox" aria-label="Select all">
+                <div class="email-cell check">
+
                 </div>
```

Comment:
Nit. Eliminate unnecessary whitespace.

```suggestion
                <div class="email-cell check"></div>
```